DSE_6.0_Release_Notes.md

```diff
@@ -1,6 +1,71 @@
 # Release notes for DataStax Enterprise 6.0
 DSE 6.0.x is compatible with Apache Cassandra&trade; 3.11 and adds additional production-certified changes, if any. Components that are indicated with an asterisk (&ast;) (if any) are known to be updated since the prior patch version.
 
+# Release notes for 6.0.17
+17 February 2022
```

Comment:
This date is yesterday. The release will go out today, right? So we should update this date.